README.md

```diff
@@ -6,14 +6,34 @@
 [![Coverage Status](https://coveralls.io/repos/github/aws/karpenter-core/badge.svg?branch=main)](https://coveralls.io/github/aws/karpenter-core?branch=main)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/aws/karpenter-core/issues)
 
+# Karpenter
+
 Karpenter improves the efficiency and cost of running workloads on Kubernetes clusters by:
 
 * **Watching** for pods that the Kubernetes scheduler has marked as unschedulable
 * **Evaluating** scheduling constraints (resource requests, nodeselectors, affinities, tolerations, and topology spread constraints) requested by the pods
 * **Provisioning** nodes that meet the requirements of the pods
 * **Removing** the nodes when the nodes are no longer needed
 
-Come discuss Karpenter in the [#karpenter](https://kubernetes.slack.com/archives/C02SFFZSA2K) channel in the [Kubernetes slack](https://slack.k8s.io/)!
+## Community, discussion, contribution, and support
+
+If you have any questions or want to get the latest project news, you can connect with us in the following ways:
+- __Using and Deploying Karpenter?__ Reach out in the [#karpenter] channel in the [Kubernetes slack] to ask questions about configuring or troubleshooting Karpenter.
+- __Contributing to or Developing with Karpenter?__ Join the [#karpenter-dev] channel in the [Kubernetes slack] to ask in-depth questions about contribution or to get involved in design discussions.
+- Join our alternating working group meetings where we share the latest project updates, answer questions, and triage issues:
+  - Bi-weekly meetings alternating between Mondays @ 9:00 PT ([convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=Seattle)) and Thursdays @ 15:00 PT ([convert to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=Seattle)) on [Chime](chimeMeeting)
+  - Invites are managed through our [Calendar](calendarInvite)
```

Comment:
Placeholders here too (chime and calendar)?

Reply:
Updated to inline them